espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/CreateSession.java

```diff
@@ -20,42 +17,24 @@
 
 public class CreateSession implements RequestHandler {
 
-    public BaseResponse handle(NanoHTTPD.IHTTPSession session, Map<String, String> uriParams) {
+    public BaseResponse handle(NanoHTTPD.IHTTPSession session, Map<String, Object> params) {
```

Comment:
It would be good to add \@Override annotation here and in all other descendant classes